src/services/mapService.tsx

```diff
@@ -5,6 +5,7 @@ import { IAPIRegionDatetimes, IRegionDatetime } from '../interfaces/api/IRegionD
 
 var baseUrl = process.env.REACT_APP_API_URL
 
+//when the example value is used in the env (if none provided http://localhost:8000/ will be used) the baseUrl is not undefined and a faulty endpoint is used to make the fetches
```

Comment:
for which reason did you add this comment? I don't really get what you mean by "faulty endpoint"